lib/sql_footprint/sql_anonymizer.rb

```diff
@@ -3,6 +3,7 @@ class SqlAnonymizer
     GSUBS = {
       /\sIN\s\((.*)\)/ => ' IN (values-redacted)'.freeze, # IN clauses
       /\s\=\s([0-9]+)/ => ' = number-redacted'.freeze, # numbers
+      /\s\!\=\s([0-9]+)/ => ' != number-redacted'.freeze, # numbers w/ not equals
```

Comment:
@ryanzbone I like the idea. I think you still have a bug or two in your regex. It wouldn't capture `<`, or `>`. But, I think this regex might work better: `/\s+(!=|=|<|>|<=|>=)\s+[0-9]+/. It's a little more explicit.